scripts/setup-ubuntu.sh

```diff
@@ -109,10 +109,24 @@ function install_fbthrift {
 }
 
 function install_conda {
-  mkdir -p conda && cd conda
-  wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
-  MINICONDA_PATH=/opt/miniconda-for-velox
-  bash Miniconda3-latest-Linux-x86_64.sh -b -p $MINICONDA_PATH
+  MINICONDA_PATH=/opt/miniconda-for-velox-jasong
+  if [ -d "${MINICONDA_PATH}" ]; then
+    if prompt "${MINICONDA_PATH} already exists. Delete?"; then
```

Comment:
Please leave out prompt from scripts that are used for automation. Also please remove personal paths.